components/settings/Account/Usage.tsx

```diff
@@ -0,0 +1,102 @@
+import { UsageByService } from "@3rdweb-sdk/react/hooks/useApi";
+import { SimpleGrid, Spinner, VStack } from "@chakra-ui/react";
+import { UsageCard } from "./UsageCard";
+import { useMemo } from "react";
+import { Heading } from "tw-components";
+
+interface UsageProps {
+  usage: UsageByService | undefined;
+  usageLoading: boolean;
+}
+
+export const Usage: React.FC<UsageProps> = ({ usage, usageLoading }) => {
+  const bundlerTotal = useMemo(() => {
+    if (!usage?.bundler || usage.bundler.length === 0) {
+      return <i>N/A</i>;
+    }
+    const total = usage.bundler?.[0].billableUsd || 0;
+
+    return new Intl.NumberFormat(undefined, {
+      style: "currency",
+      currency: "USD",
+    }).format(total);
+  }, [usage]);
+
+  return (
+    <VStack gap={8} w="full">
+      {usageLoading && <Spinner size="sm" />}
+      {!usageLoading && (
+        <VStack w="full" gap={12}>
+          {/* <VStack alignItems="flex-start" gap={6} w="full">
```

Comment:
Let's remove commented code if we don't need it

Reply:
its coming soon, this week